src/main/webapp/search-style.css

```diff
@@ -0,0 +1,42 @@
+.hidden {
```

Comment:
The whole file is a bit wonky in terms of formatting (not alphabetical either). Is Prettier not running on css files or was it causing issues?
Not something for this PR but something I'd like to know about for future use in stuff like this.

Reply:
I'll go through and fix the alphabetical order and see what I can do about the spacing. Sorry about that - I think it's a combination of me being in a rush and Prettier being weird.